integration-tests/tests/src/hooks/open-realm-before.ts

```diff
@@ -69,9 +69,10 @@ export function openRealmHook(config: OpenRealmConfiguration = {}) {
 export function closeThisRealm(this: RealmContext & Mocha.Context): void {
   if (this.closeRealm) {
     this.closeRealm({ clearTestState: true, deleteFile: true });
+  } else {
+    // Clearing the test state to ensure the sync session gets completely reset and nothing is cached between tests
+    Realm.clearTestState();
   }
-  // Clearing the test state to ensure the sync session gets completely reset and nothing is cached between tests
-  Realm.clearTestState();
```

Comment:
What's the idea behind this change? Is this not needed if the Realm can be closed?

Reply:
`this.closeRealm({ clearTestState: true, deleteFile: true });` is already calling `Realm.clearTestState`, so if we'd just call it twice otherwise